src/combine.py

```diff
@@ -267,35 +275,35 @@ def parseMp4boxMediaInfo(file_name, mp4box_path, regex_mp4box_duration):
 
 #
 # Locates the mp4box executable and returns a full path to it
-def findMp4Box(path_to_gpac_install=None, working_dir=None):
+def findMp4Box(path_to_gpac_install=None, working_dir=None, mpbox_filename=MP4BOX_DEFAULT_FILENAME):
   
-  if( not path_to_gpac_install is None and os.path.isfile(os.path.join(path_to_gpac_install, "mp4box.exe")) ):
-    return os.path.join(path_to_gpac_install, "mp4box.exe")
+  if( not path_to_gpac_install is None and os.path.isfile(os.path.join(path_to_gpac_install, mpbox_filename)) ):
+    return os.path.join(path_to_gpac_install, mpbox_filename)
 
   # Attempts to search for it under the bin folder
-  bin_dist = os.path.join(working_dir, "..\\bin\\GPAC\\mp4box.exe")
+  bin_dist = os.path.join(working_dir, f"..\\bin\\GPAC\\{mpbox_filename}")
   if( os.path.isfile(bin_dist)):
     return str(Path(bin_dist).resolve())
   
   # Attempts to search for it under C:\Program Files\GPAC
-  if( os.path.isfile("C:\\Program Files\\GPAC\\mp4box.exe")):
-    return "C:\\Program Files\\GPAC\\mp4box.exe"
+  if( os.path.isfile(f"C:\\Program Files\\GPAC\\{mpbox_filename}")):
+    return f"C:\\Program Files\\GPAC\\{mpbox_filename}"
   
   # For 32 bit installs
-  if( os.path.isfile("C:\\Program Files\\GPAC\\mp4box.exe")):
-    return "C:\\Program Files (x86)\\GPAC\\mp4box.exe"
+  if( os.path.isfile(f"C:\\Program Files\\GPAC\\{mpbox_filename}")):
+    return f"C:\\Program Files (x86)\\GPAC\\{mpbox_filename}"
   
   # Throw an error
   raise ValueError('Could not locate GPAC install, please use the --gpac switch to specify the path to the mp4box.exe file on your system.')
 
 #
 # Locates the ffmpeg executable and returns a full path to it
-def findffmpeg(path_to_ffmpeg_install=None, working_dir=None):
-  if( not path_to_ffmpeg_install is None and os.path.isfile(os.path.join(path_to_ffmpeg_install, "ffmpeg.exe")) ):
-    return os.path.join(path_to_ffmpeg_install, "ffmpeg.exe")
+def findffmpeg(path_to_ffmpeg_install=None, working_dir=None, ffmpeg_filename="ffmpeg"):
```

Comment:
Could you please make the same type of constant should be made for FFMPEG as you've done for mp4box otherwise the script breaks on Windows